src/main/java/ladder/creator/LadderCreator.java

```diff
@@ -0,0 +1,60 @@
+package ladder.creator;
+
+import ladder.Line;
+import ladder.Row;
+import ladder.constants.ExceptionMessage;
+import ladder.wrapper.unsignedInt;
+
+public abstract class LadderCreator {
+    Row[] rows;
+
+    LadderCreator(unsignedInt row, unsignedInt column) {
+        validateLadder(row, column);
+
+        this.rows = new Row[row.getValue()];
+
+        for(int i=0;i<row.getValue();i++){
+            rows[i] = new Row(column.getValue());
+        }
+    }
+
+    private void validateLadder(unsignedInt row, unsignedInt column) {
+        if(row.getValue() <= 2 || column.getValue() <= 2){
+            throw new IllegalArgumentException(ExceptionMessage.INVALID_LADDER.getMessage());
+        }
+    }
+
+    public void drawLine (Line line) {
+
+        validateLineInformation(line);
+
+        rows[line.getRow()].drawLineAt(unsignedInt.from(line.getColumn()), line.getDirection());
+    }
+
+    private void validateLineInformation(Line line) {
+        if(invalidLine(line)){
+            throw new IllegalArgumentException(ExceptionMessage.INVALID_DRAW_POSITION.getMessage());
+        }
+    }
+
+    private boolean invalidLine(Line line) {
+        if(isInvalidRow(line) || isInvalidColumn(line)){
+            return true;
+        } else {
+            return false;
+        }
+    }
```

Comment:
바로 이런 경우에도 조건문을 바로 리턴하면 깔끔할 것 같아요.

```suggestion
    private boolean invalidLine(Line line) {
        return isInvalidRow(line) || isInvalidColumn(line)
    }